.github/workflows/publish-pull-requests.yml

```diff
@@ -0,0 +1,108 @@
+name: Publish Results from Pull Requests
+
+on:
+  workflow_run:
+    workflows: [Pull Request Build]
+    types:
+      - completed
+
+permissions:
+  checks: write
+  statuses: write
+
+jobs:
+  publish:
+    # only run in the official pmd repo, where we have access to the secrets
+    if: github.repository == 'pmd/pmd'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Download PR info
+        uses: actions/download-artifact@v4
+        with:
+          name: pr-info
+          run-id: ${{ github.event.workflow_run.id }}
+      - run: |
+          PR_NUMBER="$(cat ./PR_NUMBER)"
+          echo "PR_NUMBER=${PR_NUMBER}" >> "$GITHUB_ENV"
+          PR_SHA="$(cat ./PR_SHA)"
+          echo "PR_SHA=${PR_SHA}" >> "$GITHUB_ENV"
```

Comment:
This actually would need to be validated. Need to test "gh pr view" instead, see https://github.com/orgs/community/discussions/25220#discussioncomment-11316244